2-mandatory/3-financial-times.js

```diff
@@ -14,24 +14,38 @@ function potentialHeadlines(allArticleTitles) {
     (you can assume words will always be seperated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
+    let shortestTitle = allArticleTitles[0];
+    allArticleTitles
+    .forEach((element, index) => {shortestTitle.length > element.length ? shortestTitle = element : 0;
```

Comment:
The `index` variable is not used, you can removed it.

Reply:
This comment refers to the previous version of the code (it shows 'outdated' at the top), I will see the updated version when I start a new review.